app/controllers/api/v1/messages_controller.rb

```diff
@@ -12,6 +13,10 @@ def index
       def conversation
         @conversation ||= current_user.conversations.find(params[:conversation_id])
       end
+
+      def mark_messages_as_read
```

Comment:
This should be a private method

Reply:
`mark_messages_as_read` is already a private method.